src/coreclr/jit/lower.cpp

```diff
@@ -3101,19 +3101,11 @@ void Lowering::LowerStoreLocCommon(GenTreeLclVarCommon* lclStore)
     if ((varTypeUsesFloatReg(lclStore) != varTypeUsesFloatReg(src)) && !lclStore->IsPhiDefn() &&
```

Comment:
Do we still need ` !lclStore->IsPhiDefn()` now that `PHI`s are no more in the backend?

Reply:
Nop, thanks, will delete.